app/assets/stylesheets/calendar.scss

```diff
@@ -0,0 +1,50 @@
+.calendar{
```

Comment:
As a general comment for this file, I usually prefer ordering CSS properties alphabetically.

Until we as a team can come up with some guidelines for how to group CSS properties together by 'type', which I agree is better in theory, it's what I've been going with for now to at least achieve consistency.